selenium_tests/pages.py

```diff
@@ -32,6 +39,26 @@ def footer(self) -> 'FooterElement':
     def navbar(self) -> 'NavbarElement':
         return NavbarElement(self.driver)
 
+class HomePage(BasePage):
+
+    banner_area_locator = (By.CSS_SELECTOR, "section[class='banner-holder']")
+    banner_image_locator = (By.CSS_SELECTOR, "img[alt='An image']")
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+
+    def has_banner(self):
+        banner = self.driver.find_element(*self.banner_area_locator)
+        return (
+            banner.size['width'] > 0
```

Comment:
This code looks familiar - from ArticlePage. This snippet probably needs to be extracted into its own method or function, because I see the need to check if an image is present in many other situations.